src/rateLimiter.js

```diff
@@ -60,6 +64,28 @@ RateLimiter.prototype.shouldSend = function(item, now) {
     this.perMinCounter = 0;
   }
 
+  if (item._isUncaught) {
+    // var item = {
+    //   message: message,
+    //   err: err,
+    //   custom: custom,
+    //   timestamp: now(),
+    //   callback: callback,
+    //   uuid: uuid4()
+    // };
+    var exceptionMessage = item.message;
+    var exception = item.err;
+    var exceptionHash = exceptionMessage + '-' + (exception.stack || String(exception));
```

Comment:
Because we are checking `item._isUncaught` I am guessing that `item.err == exception` is therefore not undefined here. However, `item.message` would then always be undefined so that this line would throw. If message was not undefined then exception probably would be so that `exception.stack` would throw. This line needs some care, assume that everything could be undefined and just add some conditionals, something like

```
var exceptionHash = (exceptionMessage || '') + '-' + ((exception && (exception.stack || String(exception))) || '');
```

I'm not exactly sure which of those parentheses are superfluous if any